.github/workflows/ansible-test-windows.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - windows-2019
+          - windows-2022
```

Comment:
This might be adding unnecessary breaks (last time I checked) due to the Windows 2022 image not supporting WSL v1 which the workflow relies on at this time (https://github.com/actions/runner-images/issues/5166#issuecomment-1150742571)

Reply:
> This might be adding unnecessary breaks (last time I checked) due to the Windows 2022 image not supporting WSL v1 which the workflow relies on at this time (https://github.com/actions/runner-images/issues/5166#issuecomment-1150742571) 

I can revert that change.  I was seeing different behavior in Powershell on my workstation vs the Github runner.  So I just wanted to rule that out.

Reply:
Yeah, went back and I do see now a fix for the vampire WSL action has been made! I'll be merging that change via https://github.com/lowlydba/lowlydba.sqlserver/pull/88 so you can keep it as is if you want